test/passing/tests/polytypes-janestreet.ml.ref

```diff
@@ -1,26 +1,26 @@
 let t1 : 'a 'b. 'a t -> b t = ()
 
-let t2 :
-      'a 'b.
-      'a t________________________________ -> 'b t_______________________________________
+let t2
+  : 'a 'b.
+  'a t________________________________ -> 'b t_______________________________________
```

Comment:
I do not consider this a regression as it's the formatting used in slightly different cases.